src/name.rs

```diff
@@ -511,7 +511,7 @@ fn presented_dns_id_matches_dns_id_constraint(
 //
 // Name constraints ///////////////////////////////////////////////////////////
 //
-// This is all RFC 5280 has to say about DNSName constraints:
+// This is all RFC 5280 has to say about dNSName constraints:
```

Comment:
```suggestion
// This is all RFC 5280 has to say about DnsName constraints:
```

Reply:
It is confusing, but RFC 5280 consistently uses the capitalization "dNSName".

Reply:
Today I learned.
Sorry for the noise then.